nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -111,6 +111,30 @@ let
     }
   ''));
 
+  accessLogString = { file, format, buffer, gzip, flush, off ? [], ... }:
+    (if off then "
+      access_log off;"
+    else "
+      access_log "
+        + optionalString (file != null) "${file} "
+        + optionalString (buffer != null || gzip != null || (buffer != null && flush != null)) "${format} "
+        + optionalString (buffer != null) "buffer=${buffer} "
+        + optionalString (gzip == 1) "gzip "
+        + optionalString (gzip != 1 && gzip != null) "gzip=${toString gzip} "
+        + optionalString (buffer != null && flush != null) "flush=${flush} "
+        + ";"
+      );
+
+  errorLogString = { file, logLevel, stderr ? [], ... }:
+    (if stderr then "
```

Comment:
stderr is a list and is here treated like a boolean, or am I missing something?

Reply:
This is a record of string. Example resulat ing nginx.conf:
```
 server {
    listen 0.0.0.0:80 ;
    server_name example.com ;
    error_log stderr;
```
From nginx documentation:
`
The first parameter defines a file that will store the log. The special value stderr selects the standard error file.
`